updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single operation using the web console or the OpenShift CLI (`oc`).
+Platform administrators are automatically notified when an update is available for their cluster.
+Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
+
+The {product-title} update service builds a graph of upgrade possibilities based on release images in the registry.
```

Comment:
Should this be "the OpenShift Update Service (OSUS)"? And I would remove the hard wraps here as well.

```suggestion
The OpenShift Update Service (OSUS) builds a graph of upgrade possibilities based on release images in the registry.
```

Also, if you end up making ^^this change, then you probably don't need to spell out OSUS under the CVO definition. I hope that makes sense. 😛